Fix: #105. No trailers. (#117)
ietf-wg-httpapi/ratelimit-headers: draft-ietf-httpapi-ratelimit-headers.md

   RateLimit-Policy: 10;w=1;burst=1000, 1000;w=3600
~~~

> This field cannot apper in a trailer section.

Reviewer: ```suggestion
This field cannot appear in a trailer section.
```

    RateLimit-Remaining field and RateLimit-Reset field for defining the throttling
    behavior.

15. Can I use RateLimit fields in trailers?
    Servers usually establish whether the request is in-quota before creating a response, so the RateLimit field values should be already available in that moment.
    Supporting trailers has the only advantage that allows to provide more up-to-date information to the client in case of slow responses.

>     OTOH this complicates client implementation with respect to combining fields and managing intermediaries that drop trailers.

Reviewer: ```suggestion
    However, this complicates client implementations with respect to combining fields from headers and accounting for intermediaries that drop trailers.
```
